src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    public abstract double getArea();
+
+    String toString();
```

Comment:
let's create a separate interface Drawable with method draw()
and make Figure class implement 2 interfaces, because the purposes of the interfaces are different: 1st calculates the area, second is printing out whole information

Reply:
I am not curtain how to draw all info with draw(), without passing it to overriden toString() method.